test_util/marathon.py

```diff
@@ -9,7 +9,17 @@
 
 import test_util.helpers
 
+DEFAULT_API_BASE = '/marathon/v2'
 TEST_APP_NAME_FMT = '/integration-test-{}'
+REQUIRED_HEADERS = {'Accept': 'application/json, text/plain, */*'}
+
+
+def update_headers(headers):
+    if headers:
```

Comment:
the point of this function is just to handle the repeated case where default_headers=None so you don't have to rewrite:
```
if default_headers is None:
    default_headers = {}
```
but its not much of a savings